cli/lib/kontena/callbacks/master/deploy/50_authenticate_after_deploy.rb

```diff
@@ -49,10 +49,15 @@ def after
         end
 
         require 'shellwords'
-        cmd = "master login --no-login-info --skip-grid-auto-select --verbose --name #{command.result[:name].shellescape} --code #{command.result[:code].shellescape} #{new_master.url.shellescape}"
+        cmd = [
+          'master', 'login', '--no-login-info' ,'--skip-grid-auto-select', '--verbose',
+          '--name', command.result[:name],
+          '--code', command.result[:code],
+          new_master.url
+        ]
         Retriable.retriable do
           ENV["DEBUG"] && $stderr.puts("Running: #{cmd}")
-          Kontena.run(cmd)
+          raise "retry" unless Kontena.run(cmd, returning: :status).zero?
```

Comment:
Most of these callers seem to be of the form `fail unless Kontena.run(..., :returning => :status).zero?`. Would the default `Kontena.run(cmd)` case behave identically, given that it will raise an exception, and thus retry?